config/routes.rb

```diff
@@ -13,8 +13,8 @@
   get '/admin', to: 'admins#dashboard' 
   shallow do 
     namespace :admin do 
```

Comment:
I think the syntax `namespace :admin, shallow: true do` would also work here to extract one layer of nesting